src/Services/FHIR/Observation/FhirObservationVitalsService.php

```diff
@@ -664,6 +664,12 @@ private function columnHasPositiveFloatValue($column, $record)
 
     private function populateBloodPressurePanel(FHIRObservation $observation, $dataRecord)
     {
+            // Based on conversations with Jerry Padget and Brady Miller on August 14th 2021 we decided that if the values
+            // were both 0 for bpd and bps we would treat this as a data absent reason.  In this case an attempt was made
+            // to record the data but no value was recorded (such as the blood pressure cuff becoming loose).
+        if ($dataRecord['bpd'] == 0 && $dataRecord['bps'] == 0) {
+            $observation->setDataAbsentReason(UtilsService::createDataAbsentUnknownCodeableConcept());
```

Comment:
yes!! the 0 hack!
![2DV-2](https://user-images.githubusercontent.com/278968/129462804-06af6e9a-e46e-40fc-814a-1e6a71d50eb8.gif)

Reply:
I'm going to have to find a sad face emoji.